server/controllers/Project.js

```diff
@@ -0,0 +1,96 @@
+const { User, Project } = require('../models');
+const formidable = require('formidable');
+const upload = require('../services/file-upload');
+const singleUpload = upload.single('image');
+const multiUpload = upload.array('images');
+
+
+exports.imageUpload = (req, res) => {
+    singleUpload(req, res, function (err) {
+        if (err) return res.status(422).json({ errors: [{ title: 'File Upload Error', detail: err.message }] });
+        return res.json({ 'imageUrl': req.file.location })
+    })
+}
+
+exports.projectById = (req, res, next, id) => {
+    Project.findById(id)
+        .populate('user')
+        .exec((err, project) => {
+            if (err || !project) {
+                return res.status(400).json({
+                    error: 'Project not found.'
+                });
+            }
+            req.project = project;
+            next();
+        })
+}
+
+exports.getUserProjects = (req, res) => {
+    // console.log(req.profile._id)
+    // Project.find({ user: req.profile._id })
+    //     .exec((err, projects) => {
+    //         if (err) {
+    //             return res.status(400).json({
+    //                 error: 'Projects not found.'
+    //             });
+    //         }
+    //         res.json(projects)
+    //     })
+    User.findById(req.profile._id)
+        .populate('projects')
```

Comment:
Yes this is what i'm looking for - however I wanted this in the get user route (I commented about it below). The idea was we don't really need this route, but we need a route to get the users and the projects together